scripts/generate_demo_list.js

```diff
@@ -51,6 +51,16 @@ function isDirectory(source) {
   return fs.lstatSync(source).isDirectory();
 }
 
+/**
+ * @param {string} val
+ * @returns {string}
+ */
+function encodeHtmlAttributeValue(val) {
+  return val
+    .replace(/&/g, "&amp;")
+    .replace(/"/g, "&quot;");
+}
```

Comment:
Don't we need to also handle ">" and "<", and maybe other special entities ?

Reply:
This code is only used to encode attribute values so things like `<Element myvalue="<>&amp;" />`.
In that context, it doesn't seem necessary to handle ">" and "<" or others.

From reading the HTML living standard (https://html.spec.whatwg.org/):
> In [the HTML syntax](https://html.spec.whatwg.org/#syntax), authors need only remember to use U+0022 QUOTATION MARK characters (") to wrap the attribute contents and then to escape all U+0026 AMPERSAND (&) and U+0022 QUOTATION MARK (") characters, and to specify the [sandbox](https://html.spec.whatwg.org/#attr-iframe-sandbox) attribute, to ensure safe embedding of content. (And remember to escape ampersands before quotation marks, to ensure quotation marks become &quot; and not &amp;quot;.)

_(though confusingly, https://html.spec.whatwg.org/#serialising-html-fragments, also talks about `U+00A0 NO-BREAK SPACE` which should be replaced by `&nbsp;`)_

Some literature on the subject (https://wonko.com/post/html-escaping) seem to tell that this might not be enough however, starting with the assumption that a developer might make a typo and use a still functional `'`, even sometimes "\`" or no quote at all to delimitate attribute values, in which cases, much more characters have to be escaped.
I don't know what to do with that for now. Do you think we should be more zealous? In that case I'm not sure of what to do and if it would make sense to use the `html-entities` dependency, which we use for constituting the HTML body, also for attributes.

Reply:
Maybe a solution could be to look at what other type of HTML generators are doing in that case.

Reply:
From reading a little more, it seems that using all entities is perfectly fine in attributes too.
In that case, I think I'll use the `html-entities` there too.

Reply:
Done by replacing all calls to `encodeHtmlAttributeValue` to `html-entities`'s `encode` method - which handles all entities.